### DISCUSSION TITLE
usInifiniuteQuery Doesnt cancel the query using cancelQueries()

### DISCUSSION THREAD

**Discussion by cubecleveland** [TanStack/query#4390] [General]
Hello using useInfnfinteQuery and cancelQuerioes to cancel specific queries...

The feature seem to not work with Infinitebut works with regular useQuery.. any one know wnaything about it as an open issue ?
Running react-query 3/34/0 tried both the old cancel method and the new method by passing the AborSignal

Here is our fecth code

 const id = setTimeout(() => controller.abort(), 10000);
    try {
      const response = await fetch(url, {
        method: "POST",
        credentials: "include",
        body: JSON.stringify(input),
        signal,
      });
      clearTimeout(id);
      if (!response.ok) {
        reject(new Error("Problem fetching data"));
      }
      const data = await response.json();

      resolve(data);
    } catch (ex) {
     
    }
  });
  promise.cancel = () => {
    controller.abort();
  };
  return promise;

**Comment by TkDodo** [comment]
Should work with the signal. Please show a codesandbox reproduction.
